### PR TITLE
Update and improve README + config script

### PR DESCRIPTION
**Pull request by nicolasvasilache**

### CODE REVIEW

**Comment by nicolasvasilache** [comment]
Also @giuseros, my shell is:
```
ps -p $$ 
    PID TTY          TIME CMD
2386742 pts/19   00:00:00 bash
```

**Review by stellaraccident — APPROVED** [review]
Fwiw - in these kinds of early states, I am fine with post review on basic bootstrapping stuff.

**Comment by stellaraccident** [comment]
Also, during bootstrapping, I'm fine landing before the CI is finished --
as long as it gets to the part where it is compiling. It takes an hour or
so when the llvm version bumps. During regular dev, it hasn't been so bad,
but a real pita if iterating on the config.

**Comment by nicolasvasilache** [comment]
SG, thanks for the guidance @stellaraccident !

File: configure.py [hunk 1]
```diff
@@ -110,7 +121,7 @@ def main(args):
       f"-S{os.path.join(llvm_path, 'llvm')}",
       "-DLLVM_ENABLE_LLD=ON",
```
Comment (giuseros) [inline configure.py:122]: @nicolasvasilache could you remove this since you introduced an option to specify `DLLVM_ENABLE_LLD` ?

**Review by giuseros — COMMENTED** [review]
I don't know exactly what you did, but everything seems to work now! I am happy with this bar the `lld` thing which is still enabled by default (see comments).

File: configure.py (hunk 1 above)
Comment (nicolasvasilache) [reply configure.py:122]: ah yes, sorry, see #21 
you should also have an invite in  https://github.com/google/iree-llvm-sandbox/invitations so I can add you to reviews
please also send me the github logins of a few folks I should add on your end

**Pull request merged**